pkg/executor/benchmark_test.go

```diff
@@ -1350,6 +1372,20 @@ func prepareMergeJoinExec(tc *mergeJoinTestCase, joinSchema *expression.Schema,
 		isOuterJoin:  false,
 	}
 
+	var usedIdx [][]int
+	if tc.childrenUsedSchema != nil {
+		usedIdx = make([][]int, 0, len(tc.childrenUsedSchema))
+		for _, childSchema := range tc.childrenUsedSchema {
+			used := make([]int, 0, len(childSchema))
+			for idx, one := range childSchema {
+				if one {
+					used = append(used, idx)
+				}
+			}
+			usedIdx = append(usedIdx, used)
+		}
+	}
```

Comment:
Why do we need a special one for the merge join case?

Reply:
because merge join's usedArray(bool type) is passed in the test case itself, we need an adaptor here to change it as int array type to feed the below joiner initialization